qt/widgets/instrumentview/src/InstrumentWidget.cpp

```diff
@@ -396,6 +396,45 @@ InstrumentWidgetTab *InstrumentWidget::getTab(const Tab tab) const {
     return nullptr;
 }
 
+/**
+ * @brief Get render tab from user specified tab
+ * @param tab :: render tab index
+ * @return
+ */
+InstrumentWidgetRenderTab *InstrumentWidget::getRenderTab(const Tab tab) const {
+
+  // Call to get Q widget
+  InstrumentWidgetTab *widget_tab = getTab(tab);
+
+  // Cast
+  if (widget_tab != nullptr) {
```

Comment:
This `if` statement isn't necessary See https://en.cppreference.com/w/cpp/language/dynamic_cast
```
2) If the value of expression is the null pointer value, the result is the null pointer value of type new-type.
```